client/.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "printWidth": 100,
+  "tabWidth": 2
```

```suggestion
  "tabWidth": 4
```


We can also consider "usetabs" configuration.

Also add some endOfLine configuration for consistency.